stroeer/core/v1/article.proto

```diff
@@ -961,6 +962,54 @@ message Body {
   repeated BodyNode children = 1;
 }
 
+// todo: rename to `Body`
+message BodyPart {
+  /*
+  * ### `types`
+  * | type | description |
+  * | -----| ------------|
+  * | `body` | formatted text and inlined elements |
+  * | `disclaimer` | formatted text that holds a disclaimer, e.g. used in `/gesundheit/` or `/kaufbertatung/` |
+  * | `infobox` | todo |
+  * | `table_of_contents` | todo |
+  * | `article_sources`| "Quellenapparat", list like structure containing `children[type=article_source]`. See [Samples](#article_sources). |
+  * | `article_source` | A single "Quellenapparat" item usually containing `text` or `a` |
+  */
+  string type = 1;
+  repeated BodyPart.Node children = 2;
+
+  message Node {
+    /**
+    * ### HTML `types`
+    * | type | description |
+    * | -----| ------------|
+    * | `text` | most basic `type`, its text value can be found in the `text` field. The `word_count` can be found in the `BodyNode.fields` for each `BodyNode[type=text]` |
+    * | `p` | `paragraph` / `<p>`  |
+    * | `sub_headline` | a sub headline, may be part of the _table of contents_  |
+    * | `a` | `anchor` / `<a>`  |
+    * | `strong` | `strong` / `<strong>` |
+    * | `em` | `emphasis` / `<em>` |
+    * | `br` | `line break` / `<br>` |
+    * | `ul` | `unordered list` / `<ul>` |
+    * | `ol` | `ordered list` / `<ol>` |
+    * | `li` | `list` / `<li>`  |
+    *
+    * ### Inlined custom `types`
+    * | type | description |
+    * | -----| ------------|
+    * | `img` | inline image element, check `elements` |
+    * | `video` | inline video element, check `elements` |
+    * | `gallery` | inline gallery element, check `elements` |
+    * | `oembed` | inline oEmbed element, check `elements` |
+    */
+    string type = 1;
```

Comment:
An `enum` would be perfect here. Maybe something for 1.0

Reply:
I would rather not do it here, It will become messy. This is merely a list of HTML elements, e.g. upcoming `<table>` will add drölfizig new things here like `table,td,th,tr,thead,tbody`